### DISCUSSION TITLE
Welcome to react-design-editor discussions!

### DISCUSSION THREAD

**Discussion by core7s** [core7s/react-design-editor#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pandurang-ayanworks** [comment]
Hello 
I tried using the design editor in my Next.js app. But when I try to use the useEditor hook , value that I am getting is "null".  
Thank you for your suggestions in advance.

**Reply by unrealdinesh** [reply]
Hi
is this bug still persist?
